x-pack/plugins/apm/server/routes/alerts/route.ts

```diff
@@ -34,7 +41,7 @@ const alertParamsRt = t.intersection([
   }),
   t.partial({
     groupBy: t.array(t.string),
-    kqlFilter: t.string,
+    searchConfiguration: t.string,
```

Comment:
Shouldn't this be:

```suggestion
    searchConfiguration: searchConfigurationRt,
```

You can decode stringifed json using io-ts:
https://github.com/sqren/kibana/blob/87dddd3bb2b28373814f0739ceb832cc0aa5db3e/x-pack/plugins/exploratory_view/public/routes/json_rt.ts#L24

Reply:
updated in bee1742c5c900538fe1131d9c0c0b6de350c6dcc